src/pages/ja/guides/styling.md

```diff
@@ -0,0 +1,295 @@
+---
+AstroはスタイリングやCSSの記述を簡単にするために設計されました。Astroコンポーネントの内部に直接CSSを記述したり、[Tailwind][tailwind]などのお気に入りのCSSライブラリをインポートできます。[Sass][sass]や[Less][less]などの高度なスタイリング言語もサポートされています。
+
+## Astroでのスタイリング
+
+Astroコンポーネントのスタイリングは、コンポーネントまたはページテンプレートに`<style>`タグを追加するだけで簡単です。Astroコンポーネント内に`<style>`タグを設置すると、AstroがCSSを検出し、スタイルを自動で処理します。
+
+```astro
+<style>
+  h1 { color: red; }
+</style>
+```
+
+### スタイルのスコープ
+
+Astroの`<style>`CSSルールは、**デフォルトで自動的にスコープされます**。スコープされたスタイルは、その同じコンポーネントの内部に書かれたHTMLにのみ適用されるように内部でコンパイルされます。Astroコンポーネント内に記述したCSSは、自動的にそのコンポーネントの中にカプセル化されます。
+
+```diff
+<style>
+-  h1 { color: red; }
++  h1.astro-HHNQFKH6 { color: red; }
+-  .text { color: blue; }
++  .text.astro-HHNQFKH6 { color: blue; }
+</style>
+```
+スコープされたスタイルは漏れず、サイトの他の部分に影響を与えることはありません。Astroでは、`h1 {}`や`p {}`のようなユニークではないセレクタを使用しても問題ありません。なぜなら、最終的な出力ではスコープされてコンパイルされるからです。
+
+また、スコープされたスタイルは、テンプレート内に含まれる他のAstroコンポーネントには適用されません。子コンポーネントにスタイルを設定する必要がある場合、そのコンポーネントを`<div>`(または他の要素)でラップしてからスタイルを付与するか検討してください。
+
+#### グローバルスタイル
+
+ほとんどのコンポーネントではスコープされたスタイルを使うことをおすすめしますが、いずれはグローバルでスコープされていないCSSを書くまっとうな理由が見つかるかもしれません。この場合、`<style is:global>`属性を使って、CSSの自動的なスコープを無効にできます。
+
+```html
+<style is:global>
+  /* スコープされず、ブラウザにそのまま配信されます。
+     サイト内の全ての<h1>タグに適用されます。*/
+  h1 { color: red; }
+</style>
+```
+
+また、`:global()`セレクタを使用すると、同じ`<style>`タグ内にグローバルなスタイルとスコープ付きのスタイルを混在させられます。これは、コンポーネントの子要素にスタイルを適用するための強力なパターンになります。
+
+```astro
+<style>
+  /* スコープされ、このコンポーネントに対してのみ適用される。 */
+  h1 { color: red; }
+  /* グローバルが混在、子要素の`h1`にのみ適用されます。*/
+  article :global(h1) {
+    color: blue;
+  }
+</style>
+<h1>Title</h1>
+<article><slot /></article>
+```
+
+これは、ブログの投稿や、CMSを使用したドキュメントなど、コンテンツがAstroの外にあるものをスタイルするのに最適な方法です。ただし、特定の親コンポーネントを持つかどうかで外観が異なるコンポーネントは、トラブルシューティングが困難になる可能性があるので注意が必要です。
+
+スコープされたスタイルができるだけ頻繁に使用されるべきです。グローバルなスタイルは、必要なときだけ使うべきでしょう。
+### CSS変数
+
+<Since v="0.21.0" />
+
+Astroの`<style>`は、ページ上で利用可能なあらゆるCSS変数を参照できます。また、`define:vars`ディレクティブを使用して、コンポーネントのfront-matterから直接CSS変数を渡せます。
+
+```astro
+---
+const foregroundColor = "rgb(221 243 228)";
+const backgroundColor = "rgb(24 121 78)";
+---
+<style define:vars={{ foregroundColor, backgroundColor }}>
+  h1 {
+    background-color: var(--backgroundColor);
+    color: var(--foregroundColor);
+  }
+</style>
+<h1>こんにちは</h1>
+```
+
+📚 `define:vars`については、[テンプレートディレクティブ](/ja/reference/directives-reference/#definevars)のページをご覧ください。
+
+## 外部のスタイル
+
+外部のグローバルなスタイルシートを参照する方法は2つあります。プロジェクトのソース内にあるファイルの場合はESMのインポート、`public/` ディレクトリにあるファイルやプロジェクトの外部でホストされているファイルの場合は絶対URLリンクになります。
+
+📚 詳しくは`public/`や`src/`にある[静的アセット](/ja/guides/imports/)の使い方をご覧ください。
+
+### ローカルスタイルシートのインポート
+
+> ⚠️ npmパッケージからインポートする場合、`astro.config`を更新する必要がある場合があります。以下の[スタイルシートをnpmパッケージからインポートする](#npmパッケージからスタイルシートをインポートする)をご覧ください。
+
+ESMのインポート構文を使用して、Astroコンポーネントのfront-matterでスタイルシートをインポートできます。CSSのインポートは、[Astroコンポーネント内の他のESMのインポート](/ja/core-concepts/astro-components/#コンポーネントスクリプト)のように動作し、**コンポーネントからの相対パス**を、ほかのimportと同様にコンポーネントスクリプトの**先頭**に記述しなければなりません。
```

Comment:
```suggestion
ESMのインポート構文を使用して、Astroコンポーネントのfront-matterでスタイルシートをインポートできます。CSSのインポートは、[Astroコンポーネント内の他のESMのインポート](/ja/core-concepts/astro-components/#コンポーネントスクリプト)のように動作し、**コンポーネントからの相対パス**を、ほかのインポートと同様にコンポーネントスクリプトの**先頭**に記述しなければなりません。
```

Reply:
@yuto343 Not sure if we should have applied this before merging? If you think @myakura is correct would you mind opening another quick PR with the fix? Otherwise, thanks for another page! 🚀